01/app.js

```diff
@@ -1,15 +1,19 @@
 function getProductList() {
     return [
         {name: 'JavaScript: podstawy', type: 'book', count: 3, price: 67.19, discount: 0.21},
-        {name: 'React: podstawy', type: 'book', count: 4, price: 79.17, discount: 0.27}
+        {name: 'React: podstawy', type: 'book', count: 4, price: 79.17, discount: 0.27},
+
+        
     ]
 }
 
 function getTotalPrice(productList) {
     return productList.reduce( (product, acc) => {
```

Comment:
Przydałoby się zmienić kolejność parametrów - inaczej to niepotrzebnie wprowadza w błąd.